talk/owt/BUILD.gn

```diff
@@ -498,7 +511,10 @@ if (owt_include_tests) {
 
     # Only the root target should depend on this.
     visibility = [ "//:default" ]
-    deps = [ ":owt_unittests" ]
+    deps = [
+      ":owt_unittests",
+      "//talk/owt/sdk/sample",
```

Comment:
owt_sdk_p2p target should not depend on sample.

Reply:
It it a mistake. Should be put inside `owt_sdk_test`.